starface999/day2.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+
+  <!-- set viewport to device width to allow responsiveness -->
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <!-- Bootstrap CDN -->
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+
+  <!-- custom styles (always require after Bootstrap) -->
+  <link rel="stylesheet" href="style.css">
+
+  <title>Bootstrap Boilerplate</title>
+  <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
+   <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+</head>
+ <div class="container">
+    <div class="row">
+      <div class="col-sm-6">
+        <h1>Column 1</h1>
+        <ul>
+          <li>What is the best way to start integrating keyboard shortcuts and quick commands into everyday coding?</li>
+          <li>What does a callback do?</li>
+          <li>can you please elaborate on how scope works?</li>
```

Comment:
I think you'll see more of this at least in practice going forward.  It is a key concept.
The short version is: things are only accessible where they are created.
So if your whole program is a circle, things created in that circle are available in the circle.  If you make smaller circles (functions) inside that circle they have access to both the contents of themselves and the big circle.